blackbox_repository/tabulated_benchmark.py

```diff
@@ -204,23 +223,22 @@ def blackbox(self) -> Blackbox:
     def __getstate__(self):
         # we serialize only required metadata information since the blackbox data is contained in the repository and
         # its raw data does not need to be saved.
-        return {
```

Comment:
Unless there is a particular reason, avoiding to define a custom `__getstate__` in the parent is better (we would also need to write a `__setstate__` then).

So I would suggest to do only:
```
    return {
            'seed': self.seed,
            'seed_for_trial': self._seed_for_trial,
            'blackbox_name': self.blackbox_name,
            'dataset': self.dataset,
            'surrogate': self._surrogate,
            'elapsed_time_attr': self.elapsed_time_attr,
            'time_this_resource_attr': self._time_this_resource_attr,
            'max_resource_attr': self._max_resource_attr,
            'simulatorbackend_kwargs': self.simulatorbackend_kwargs,
        }
```

Reply:
Done